nessie/api/job_controller.py

```diff
@@ -34,12 +34,30 @@
 from nessie.jobs.generate_boac_analytics import GenerateBoacAnalytics
 from nessie.jobs.generate_intermediate_tables import GenerateIntermediateTables
 from nessie.jobs.resync_canvas_snapshots import ResyncCanvasSnapshots
+from nessie.jobs.scheduling import get_scheduler
 from nessie.jobs.sync_canvas_snapshots import SyncCanvasSnapshots
 from nessie.jobs.sync_file_to_s3 import SyncFileToS3
 from nessie.lib.http import tolerant_jsonify
 from nessie.lib.metadata import update_canvas_sync_status
 
 
+@app.route('/api/job/schedule', methods=['GET'])
+def job_schedule():
+    sched = get_scheduler()
```

Comment:
Did you get a chance to try `sched.print_jobs`. Just curious because I haven't tried it myself yet...

Reply:
@raydavis I did, and the output is pretty much what you see here, only it doesn't include the job args, which is where our actual task names live.

Reply:
Yeah, probably a good idea to include the task names.